src/controllers/blocks/BlocksController.ts

```diff
@@ -92,16 +93,37 @@ export default class BlocksController extends AbstractController<BlocksService>
 		res
 	) => {
 		const eventDocsArg = eventDocs === 'true';
-		const extrsinsicDocsArg = extrinsicDocs === 'true';
+		const extrinsicDocsArg = extrinsicDocs === 'true';
 
-		const hash =
-			finalized === 'false' || !this.options.finalizes
-				? (await this.api.rpc.chain.getHeader()).hash
-				: await this.api.rpc.chain.getFinalizedHead();
+		let hash, queryFinalizedHead, omitFinalizeTag;
+
+		// If the network chain doesn't finalize blocks, we dont want a finalize tag.
+		// For example: Any Proof Of Work chain would want to omit the finalize tag
```

Comment:
Hmm, yea i guess it could technically speaking have a finality gadget, I think it would be a little redundant for a PoW, but it is definitely possible. So less is definitely more here.